pkg/eve/runme.sh

```diff
@@ -48,7 +49,7 @@ dump() {
 
 do_help() {
 cat <<__EOT__
-Usage: docker run lfedge/eve [-f fmt] version|rootfs|live|installer_raw|installer_iso|installer_net
+Usage: docker run lfedge/eve [-f fmt] [-o option] version|rootfs|live|installer_raw|installer_iso|installer_net
```

Comment:
Honestly -- I'm NOT at all a big fan of this being an option to the lfedge/eve container @giggsoff -- personally I think it sort of breaks the customization model of where the images are expected to be more or less generic and all customizations are done post producing the image.

The option also breaks orthogonality (since it isn't really applicable to non-installer images).

Therefore two questions:
   1. can we do this differently?
   2. if there's a good 'no' answer to the above -- I honestly think we should NOT document it and keep thinking about a good 'yes' answer to #1

Reply:
The same things may be done by manually modifying of grub.cfg on config partition or ipxe.cfg. So, it was added here for just convenience.
I do not fully understand the comment (the second option), do you propose to remove the comment or completely remove this feature from this PR?

Reply:
I'd rather completely remove the feature for now -- but if it is somehow going to make your workflow super difficult we can settle on removing the user-visible indication of the feature.

Reply:
I removed them for now in favor of modification of grub/ipxe manually